media/rendering.html

```diff
@@ -18,55 +18,63 @@ <h2>Well-deployed technologies</h2>
         <p data-feature="Rendering of media fragments">Users often want to share a pointer to a specific position within the timeline of a video (or an audio) feed with friends on social networks, and expect media players to jump to the requested position right away. The <a data-featureid="media-frags">Media Fragments URI</a> specification defines a syntax for constructing media fragment URIs and explains how Web browsers can use this information to render the media fragment.</p>
       </section>
       <section class="featureset in-progress">
         <h2>Specifications in progress</h2>
+        <div data-feature="Rendering in different color spaces">
+          <p>Wide-gamut displays are becoming more and more common. <a data-featureid="css-color-space/icc-colors">CSS Colors Level 4</a> adds color spaces beyond the classical sRGB so that style sheets can leverage new colors available. Similarly, the <a data-featureid="color-gamut">CSS Media Queries Level 4</a> specification includes means to detect wide-gamut displays and adapt the rendering of the application to these improved color spaces.</p>
+          <p>The <a data-featureid="mediaqueries5">CSS Media Queries Level 5</a>, to be published after Level 4, will introduce a <code>video-</code> prefix to some features to detect differences between the video plane and the graphics plane on devices that render video separately from the rest of an HTML document, and that e.g. may support High Dynamic Range (HDR) for video and only Standard Dynamic Range (SDR) for other types of content.</p>
+        </div>
+        <p data-feature="Rendering of protected media">Four extensions to Encrypted Media Extensions (EME) are under development in the Media Working Group: <a data-featureid="eme-v2/persistent-usage-record">Persistent Usage Record sessions</a>, <a data-featureid="eme-v2/hdcp-detection">HDCP Detection</a>, <a data-featureid="eme-v2/encryption-scheme">Encryption scheme capability detection</a> and an <a data-featureid="eme-v2/find-existing-sessions">API to find existing sessions</a>.</p>
         <div data-feature="Rendering of captions">
-          <p>The <a data-featureid="ttml">Timed Text Markup Language 2</a> (TTML2) specification extends TTML1 with advanced features for animations, styling, embedded content and metadata. The <a data-featureid="ttml-imsc11">IMSC1.1 profile</a>, backwards compatible with the IMSC1 profile, is based on TTML2.</p>
+          <p>The <a data-featureid="adpt">Audio Description profile</a> is a text-based profile of TTML2, intended to support audio description workflows worldwide, including description creation, script delivery and exchange and generated audio description distribution.</p>
```

Comment:
s/text-based profile of TTML2/profile of TTML2 incorporating audio features

Reply:
Done, thanks for the review!

Reply:
Thank you @tidoust !